### ISSUE TITLE
Deep learning project

### ISSUE THREAD

**Issue by DimitriDiantos** [DimitriDiantos/Deeplearning#1]
-TensorFlow Basics
-MLP, Learning rate, Overfitting, and Hyper-parameters
-CNNs, ResNets, Transfer Learning and Grad-CAM
-Anomaly Detection using Autoencoders
